sklearn/linear_model/tests/ransac_iterations.py

```diff
@@ -0,0 +1,20 @@
+from sklearn import linear_model
+from sklearn.linear_model.ransac import _dynamic_max_trials
+import numpy as np
+
+X =np.linspace(0,100,100)[:,None]
+print X.shape
+Y = 2*X
+Y[80:] = np.random.rand(20,1)
+Y[:80] += np.random.rand(80,1)
+
+for i in range(10):
+    clt_ransac = linear_model.RANSACRegressor(linear_model.LinearRegression(),max_trials=100)
```

Comment:
we'd like to use fixed random state, but I realise this test is to ensure the property is invariant to randomness. So add `random_state=i` here.

I don't know what `clt` is here. maybe just call it `reg`